check_source.py

```diff
@@ -221,10 +221,13 @@ def check_source_submission(self, source_project, source_package, source_revisio
 
         new_info = self.package_source_parse(source_project, source_package, source_revision, target_package)
         filename = new_info.get('filename', '')
-        if not (filename.endswith('.kiwi') or filename == 'Dockerfile') and new_info['name'] != target_package:
+        expected_name = target_package
+        if target_package.startswith('preinstallimage-'):
```

Comment:
Actually, checking for `filename == "preinstallimage"` might be better

Reply:
but then we're missing that the package name is starting with 'preinstallimage-'

Reply:
Yep, that has to be done separately:

```
if it builds a preinstallimage:
    make sure the source package name starts with "preinstallimage-"
    make sure the "binary" name is unset resp. "preinstallimage"
```

Without that, it would accept submitting an RPM package to `preinstallimage-foo` which had `Name: preinstallimage`